docs/manual/developer_guide.adoc

```diff
@@ -1208,6 +1208,9 @@ Available high-level Jinja macros to generate Bash remediations:
 - `bash_coredump_config_set` -  Set Coredump configuration in `/etc/systemd/coredump.conf`
 - `bash_package_install` - Install a package
 - `bash_package_remove` - Remove a package
+- `disable_prelink` - disables prelinking
```

Comment:
Question: if these are going to be bash remediations, do we want to prefix them with `bash_`? Then if we get a similar macro for Ansible, we can call it `ansible_`. I think it'd be cool if we could have a `disable_prelink` macro that'll call `ansible_disable_prelink` or `bash_disable_prelink` depending on execution context but I don't know if that's possible.  (The other issue with that is that sometimes the ansible mode might support more features than the bash version does, e.g., `lineinfile` in Ansible uses PCRE regexes which I haven't found a good way of supporting in bash without requiring Python or Perl or something and "rewriting" that portion of the Ansible module).

Reply:
Ah I see, `disable_prelink` was added in a previous PR but wasn't documented. Fixing that is out of scope for this PR then. Should I open a separate PR when this one merges and fix that for all three?

Reply:
@cipherboy great point, I agree, I didn't think about that so I kept the original name, but adding a prefix would greatly improve it. Yes, a separate PR would be great!